### ISSUE TITLE
Even more documentation...

### ISSUE THREAD

**Issue by tadly** [jdf76/plugin.video.youtube#541]
As @anxdpanic seems to be enjoying documenting things lately (which I really **really** do appreciate), some add-on settings (one in particular in my case) would be nice if they where documented/described somewhere.

Sadly kodi doesn't support extended setting descriptions making it sometimes hard to _bring the point across_ with the limited space available.

For this reason it would be nice to have it documented at least somewhere (kodi forum in one of the main posts, kodi wiki or github wiki <- would allow others to contribute)

The one I was referring to earlier is _Use playback history (watched, resume tracking)_ which has me (also dev, addon-dev, nerd... I know my stuff is what I'm saying) super puzzled as to what it does.
Others would be:
* Support alternative player
* Use youtube website urls
* Cache Size
  How does it improve stuff? I wouldn't have noticed anything but it set it to 100 none-the-less
  Does this even apply if inputstream is used?

Most stuff **is** self-explanatory but even so *could* do with some description as to "how it works" or what side-effects it may have if turned on/off etc.
_Age gate_ would be one of those.

The whole _MPEG-DASH_ (and inputstream) in general could do with some explaining (not necessarily technical, just to give a general idea) as this will actually improve many things if enabled and is something the none-technical user could be interested in (f.e. 1080p streaming).

As I seem to be writing a novel I'll leave it at that for the time being.

p.s. I'm happy to help documenting stuff once I myself understood what specific things do :)

**Comment by jmh2002** [comment]
I agree, and I have a similar mental list to what you have mentioned here, and some. I also have some empty posts reserved on the first page for documenting such things, but in recent months unfortunately I haven't had the same time as in the past to create more FAQs.

**Comment by anxdpanic** [comment]
I _hoping_ to get around to do this for this and the Twitch add-on some time soon.

Use playback history (watched, resume tracking) : The add-on tracks watched progress and resume points, instead of using Kodi's builtin tracking. It was added while watch marks were broken in Kodi 18.

Support alternative player : support players from playerfactorycore.xml
Use youtube website urls : pass 'https://www.youtube.com/watch?v=wA38GCX4Tb0' to player from playerfactorycore.xml (MPC-BE supports this, and can achieve 4K + HDR)

Cache Size : the data being cached is api results

**Comment by jmh2002** [comment]
What are your thoughts moving forward with this @anxdpanic? For ordinary users should this info still be copied over into the first forum pages like before, or oppositely should we move the info here and reply to user questions on Kodi.tv with a link to the appropriate github wiki instead?

**Comment by anxdpanic** [comment]
Long term I think moving everything to here would be ideal. We can link directly to topics that way, and we can assist each other in keeping it up to date.

The forum posts would still be great to have regardless. The posts allow the faq to be searched via the forum which would be a big loss in my eyes. Would be able to simplify the posts with references to the wiki here. 

What are your thoughts?

If either of you would like access to edit the wiki here, I'm sure @jdf76 would be happy to help out with that

**Comment by jdf76** [comment]
At some point I created a wiki account access.

You can email me if you want the credentials.

Jeff

**Comment by ginanotes** [comment]
If only I knew how this api stuff worked!
